### PR TITLE
RTL-Gaurav

### PR DESCRIPTION
**Pull request by gauravbhatt007**
### What do you want to achieve?
-RTL configuration in the chatbot conversations

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-Locally on the machine

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 
Screenshot (125)
Screenshot (117)
Screenshot (119)
Screenshot (120)
Screenshot (121)
Screenshot (122)
Screenshot (123)
Screenshot (124)

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/plugin.js [hunk 1]
```diff
@@ -145,6 +145,13 @@ function createCustomClasses(classSettings) {
     document.getElementsByTagName('head')[0].appendChild(style);
 }
 
+// Set language and direction based on browser's language
+function languagedirectionchangeauto() {
```
Comment (devashishmamgain) [inline webplugin/plugin.js:149]: use camelCase naming convention for javascript code

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

**Review by prochnost04 — DISMISSED** [review]

**Comment by prochnost04** [comment]
@gauravbhatt007 You can remove log statements

**Review by devashishmamgain — CHANGES_REQUESTED** [review]
@gauravbhatt007 please check the comment regarding camelCase
Also please test it on mobile browser as well

**Comment by devashishmamgain** [comment]
Screenshot 2024-01-14 at 11 11 10 AM
* mandatory sign should come on left
eg: https://www.jotform.com/form/203543029872052#preview

File: webplugin/scss/components/_km-rtl.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
+//USED WHEN THE DIRECTION/DIR IS LTR(RIGHT TO LEFT)
+
+[dir='rtl'] {
+    .mck-box .mck-nav-tabs {
+        padding-right: 0 !important;
+    }
+    // .mck-box{
```
Comment (devashishmamgain) [inline webplugin/scss/components/_km-rtl.scss:7]: remove all commented code, if it is not required

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/plugin.js [hunk 3]
```diff
@@ -145,6 +145,12 @@ function createCustomClasses(classSettings) {
     document.getElementsByTagName('head')[0].appendChild(style);
 }
 
+// Set language and direction based on browser's language
+function languageDirectionChangeAuto() {
+    document.documentElement.lang = navigator.language.toLowerCase();
+    document.documentElement.dir = document.documentElement.lang === 'ar' || document.documentElement.lang === 'he' ? 'rtl' : 'ltr';
```
Comment (panwaranuj01) [inline webplugin/plugin.js:151]: @gauravbhatt007 You are modifying the page direction instead of the kommunicate iframe.

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

File: webplugin/plugin.js (hunk 3 above)
Comment (gauravbhatt007) [reply webplugin/plugin.js:151]: sir i am scanning the the language and passing the direction to communicate iframe
Screenshot (115)

Comment (gauravbhatt007) [reply webplugin/plugin.js:151]: Done the required changes
Screenshot (126)

**Review by panwaranuj01 — DISMISSED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Review by prochnost04 — APPROVED** [review]

**Comment by Sanjeevsinghnegi** [comment]
@prochnost04 this rtl is based on browser lang, do we need to add support for adding rtl from widget script? eg: 
`isRtlSupport: true`
Also two languages is added "ar" and "he" what's for others rtl lang?

**Comment by devashishmamgain** [comment]
> @prochnost04 this rtl is based on browser lang, do we need to add support for adding rtl from widget script? eg: `isRtlSupport: true` Also two languages is added "ar" and "he" what's for others rtl lang?

good point @Sanjeevsinghnegi i will file separate issue for app level settings for this

**Comment by devashishmamgain** [comment]
> > @prochnost04 this rtl is based on browser lang, do we need to add support for adding rtl from widget script? eg: `isRtlSupport: true` Also two languages is added "ar" and "he" what's for others rtl lang?
> 
> good point @Sanjeevsinghnegi i will file separate issue for app level settings for this

filed https://kommunicate.atlassian.net/browse/FWH-1295 for settings

@gauravbhatt007 check the comment regarding the languages check which support RTL

**Comment by gauravbhatt007** [comment]
> @prochnost04 this rtl is based on browser lang, do we need to add support for adding rtl from widget script? eg: `isRtlSupport: true` Also two languages is added "ar" and "he" what's for others rtl lang?

sir i have added 13 languages in total now to the rtl part and tested it. It is working fine.

**Review by devashishmamgain — APPROVED** [review]

**Review by prochnost04 — APPROVED** [review]

**Pull request merged**